### PR TITLE
changes for login.php & login.css

### PR DESCRIPTION
**Pull request by RosarioAngelica**
Hello! I made a changes to our login.php & login.css. Please review it, Thank you!

### CODE REVIEW

File: MemoirVerse.UI/login.php [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-  <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>MemoirVerse Login</title>
-    <link rel="stylesheet" href="./index.css"/>
-  </head>
-  <body>
+    <link rel="stylesheet" href="login.css">
```
Comment (Trinidad-KianLee) [inline MemoirVerse.UI/login.php]: @RosarioAngelica 
can you please add "./login.css" so that it will be functional to all types of browser.

**Review by Trinidad-KianLee — CHANGES_REQUESTED** [review]

File: MemoirVerse.UI/login.php [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-          <a href="signup.php">Sign Up</a>
-        </p>
-      </div>
+        <div class="welcome-section">
+            <h1>Welcome to MemoirVerse!</h1>
+            <p>Enter your personal details to use all of site features.</p>
+        </div>
+        <div class="login-section">
+            <h2>Sign in</h2>
+            <form id="login-form">
```
Comment (Trinidad-KianLee) [inline MemoirVerse.UI/login.php]: id should be in snake case

**Review by Trinidad-KianLee — CHANGES_REQUESTED** [review]

**Comment by RosarioAngelica** [comment]
Hello I just removed the index.css as per advised and kindly check again all the codes and steps. Thank you

**Review by Trinidad-KianLee — APPROVED** [review]
looks good to me

**Comment by Escuro-JohnDenielL** [comment]
Looks good!!

**Review by Escuro-JohnDenielL — APPROVED** [review]

**Pull request merged**